core/workspace_svg.js

```diff
@@ -425,7 +425,7 @@ Blockly.WorkspaceSvg.prototype.createDom = function(opt_backgroundClass) {
     bottom = this.addTrashcan_(bottom);
   }
   if (this.options.zoomOptions && this.options.zoomOptions.controls) {
-    bottom = this.addZoomControls_(bottom);
+    this.addZoomControls_(bottom);
```

Comment:
Does `bottom` not get used after this line?

Reply:
Correct.  If we were adding more controls below (above?  I don't remember) we could use the result of addZoomControls_, but in this case we're not going to.